src/java.sql.rowset/share/classes/com/sun/rowset/providers/RIXMLProvider.java

```diff
@@ -236,7 +236,7 @@ public String getVersion() {
     }
 
     /**
-     * Returns the vendor name of the Reference Implemntation Optimistic
+     * Returns the vendor name of the Reference Implementation Optimistic
```

Comment:
L240: an optimistic _what_ provider? :-)

Reply:
Yep typo there.  The original authors needed to run spell check ;-)